### PR TITLE
IA-2208: [API] Create migration for feature flags

### PR DESCRIPTION
**Pull request by hakifran**
Explain what problem this PR is resolving
- Create migration for feature flags
Related JIRA tickets : [IA-2208](https://bluesquare.atlassian.net/browse/IA-2208)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [x] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes
- Add migration for creating new feature flags

## Notes

[IA-2208]: https://bluesquare.atlassian.net/browse/IA-2208?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: iaso/migrations/0217_add_feature_flag_MOBILE_CHECK_FORMS_UPDATE.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.15 on 2023-06-22 16:00
+
+from django.db import migrations
+
+
+def create_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.create(
+        code="MOBILE_CHECK_FORMS_UPDATE",
+        name="Mobile: Check if the user can update a form.",
```
Comment (bmonjoie) [inline iaso/migrations/0217_add_feature_flag_MOBILE_CHECK_FORMS_UPDATE.py]: The description should be: 
> Mobile: Warn the user when forms have been updated

File: iaso/migrations/0218_add_feature_flag_MOBILE_CHECK_OU_UPDATE.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.15 on 2023-06-22 16:11
+
+from django.db import migrations
+
+
+def create_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.create(
+        code="MOBILE_CHECK_OU_UPDATE",
+        name="Mobile: Check if the user can update an orgUnit.",
```
Comment (bmonjoie) [inline iaso/migrations/0218_add_feature_flag_MOBILE_CHECK_OU_UPDATE.py]: The description should be: 
> Mobile: Warn the user when the Org Units have been updated

File: iaso/migrations/0219_add_feature_flag_MOBILE_FORCE_FORMS_UPDATE.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.15 on 2023-06-22 16:18
+
+from django.db import migrations
+
+
+def create_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.create(
+        code="MOBILE_FORCE_FORMS_UPDATE",
+        name="Mobile: Check if the user can force update a form.",
```
Comment (bmonjoie) [inline iaso/migrations/0219_add_feature_flag_MOBILE_FORCE_FORMS_UPDATE.py]: The description should be: 
> Mobile: Warn the user when forms have been updated and force them to update

File: iaso/migrations/0220_add_feature_flag_MOBILE_FORCE_OU_UPDATE.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.15 on 2023-06-22 16:21
+
+from django.db import migrations
+
+
+def create_feature_flags(apps, schema_editor):
+    FeatureFlag = apps.get_model("iaso", "FeatureFlag")
+    FeatureFlag.objects.create(
+        code="MOBILE_FORCE_OU_UPDATE",
+        name="Mobile: Check if the user can force update an orgUnit.",
```
Comment (bmonjoie) [inline iaso/migrations/0220_add_feature_flag_MOBILE_FORCE_OU_UPDATE.py]: The description should be: 
> Mobile: Warn the user when the Org Units have been updated and force them to update

**Review by bmonjoie — CHANGES_REQUESTED** [review]

**Review by bmonjoie — APPROVED** [review]

**Comment by olethanh** [comment]
that look ok to me. I merged with main and pushed a merge migration to resolve conflicts.

But I'm wondering if we should not handle the case for when the feature flag already exists if they were already created manually on some of the environments

**Pull request merged**